app/src/test/java/ua/test/mega/tester/junit/OrderManagementProcessorTest.java

```diff
@@ -0,0 +1,123 @@
+package ua.test.mega.tester.junit;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit4.SpringRunner;
+import ua.test.mega.tester.adapters.*;
+import ua.test.mega.tester.core.OrderManagementProcessor;
+import ua.test.mega.tester.core.api.*;
+import ua.test.mega.tester.core.api.model.Currency;
+import ua.test.mega.tester.core.api.model.Order;
+import ua.test.mega.tester.core.api.model.Side;
+import ua.test.mega.tester.core.exceptions.InconsistentOrderAmount;
+import ua.test.mega.tester.core.exceptions.InconsistentOrderParameters;
+
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+
+@RunWith(SpringRunner.class)
+public class OrderManagementProcessorTest {
+    private static UserAdapter userAdapter;
+    private OrderManagementProcessor orderManagementProcessor;
+    private Order order;
+    private Order actual;
```

Comment:
I did not catch why variables have these names

Reply:
fixed